sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -896,6 +897,74 @@ The following add-on capabilities are available in this SDK:
 
 Note that some add-on capabilities will incur additional charges. See pricing: https://azure.microsoft.com/pricing/details/ai-document-intelligence/.
 
+### Get Raw JSON Result
+
+Can get the HTTP response by passing parameter `raw_response_hook`.
+<!-- SNIPPET:sample_get_raw_response.raw_response_hook -->
+
+```python
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documentintelligence import DocumentIntelligenceAdministrationClient
+endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+client = DocumentIntelligenceAdministrationClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+
+responses = []
+
+def callback(response):
+    response_status_code = response.http_response.status_code
+    response_body = response.http_response.json()
+    responses.append(response_status_code)
+    responses.append(response_body)
+
+with client:
+    client.get_resource_info(raw_response_hook=callback)
+
+print(f"Response status code is: {responses[0]}")
+print(f"Response body is: {responses[1]}")
+```
+
+<!-- END SNIPPET -->
+
+Also, can use the `send_request` method to send custom HTTP requests and get raw JSON result from HTTP responses.
+
+<!-- SNIPPET:sample_send_request.send_request -->
+
+```python
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.core.rest import HttpRequest
+from azure.ai.documentintelligence import DocumentIntelligenceAdministrationClient
+endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+client = DocumentIntelligenceAdministrationClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+
+with client:
+    # The `send_request` method can send custom HTTP requests that share the client's existing pipeline,
+    # Now let's use the `send_request` method to make a resource details fetching request.
+    # The URL of the request should be absolute, and append the API version used for the request.
+    request = HttpRequest(method="GET", url=f"{endpoint}/documentintelligence/info?api-version=2024-07-31-preview")
+    response = client.send_request(request)
+    response.raise_for_status()
+    response_body = response.json()
+    print(
+        f"Our resource has {response_body['customDocumentModels']['count']} custom models, "
+        f"and we can have at most {response_body['customDocumentModels']['limit']} custom models."
+        f"The quota limit for custom neural document models is {response_body['customNeuralDocumentModelBuilds']['quota']} and the resource has"
+        f"used {response_body['customNeuralDocumentModelBuilds']['used']}. The resource quota will reset on {response_body['customNeuralDocumentModelBuilds']['quotaResetDateTime']}"
+    )
```

Comment:
Should we be showcasing printing the same raw response information in both samples?

Reply:
I think it's not necessary to have the same printing in both samples. We put the two samples together, it should be easy to find how to use the raw response when cx checking this doc.

Reply:
It would be better for folks who look at the individual samples files since this content comes from those

Reply:
Updated